templates/accounts/profile.html

```diff
@@ -4,12 +4,25 @@
 
 {% block content %}
 <div class="profile">
-    <h1 class="my-5"></h1>
-    <p>{{ profile_user.username }}のプロフィール</p>
-    
+    {% if profile_user.username == user.username %}
+    <h1>My profile</h1>
     <form action="{% url 'accounts:logout' %}" method="POST">
-    {% csrf_token %}
-    <button type="submit">ログアウト</button>
+        {% csrf_token %}
+        <button type="submit">ログアウト</button>
     </form>
+
+    {% endif %}
+    <h2>過去のツイート</h2>
+
+        {% for tweet in tweets %}
+        [ツイート時間] {{tweet.created_at}}
```

Comment:
```suggestion
        [ツイート時間] {{ tweet.created_at }}
```
このように`{{` `}}` や `{%` `%}` の間は1スペース入れた方が、統一感もあって読みやすそうです（Linterが聞かない部分なのですべて手で直す必要があります）。